resources/views/tasks/index.blade.php

```diff
@@ -14,8 +14,11 @@
         __($title) => null,
     ]])
 @endsection
+
 @section('content')
-  <div class="px-3 page-content mb-0" id="tasks">
+<div id="tasks">
```

Comment:
The `div` with `id="tasks"` is repeated twice. IDs should be unique in an HTML document. This can cause issues with DOM manipulation and JavaScript behavior.

Reply:
repeated Id was removed.
Blade Formatter was applied to the entire document.